container_pipeline/workers/linter.py

```diff
@@ -33,6 +33,7 @@ def handle_job(self, job):
         """
         This menthod handles the job received for dockerfile lint worker.
         """
+        print 1/0
```

Comment:
What does this do? Wouldn't execution fail here with `ZeroDivisionError`? Rest of the function won't be executed IIUC.

Reply:
Again, you should see the commit message as why this is added.
This was added intentionally to see how console output is if something goes wrong.